packages/upshot/src/endpoint/statistics.ts

```diff
@@ -13,7 +13,7 @@ const customError = (data: ResponseSchema) => {
 }
 
 export const description =
-  'This endpoint returns statistics for an NFT collection in wei: \n 1. Floor price \n 2. Market cap \n 3. Floor price and Market cap packed as bytes32'
+  'This endpoint returns statistics for an NFT collection in wei: \n 1. Floor price \n 2. Market cap \n 3. Floor price and Market cap packed as bytes32 \n 4. Floor price and unix timestamp as bytes32'
```

Comment:
Be accurate with the description and the function names. It encodes `[timestamp, price]`, not `[price, timestamp]`. Make sure description and function names reflect that.
Also, if there are different kind of price evaluations and somehow we are differentiating them by "floor", "cap", etc. make sure functions also reflect that.